src/smdba/postgresqlgate.py

```diff
@@ -155,9 +155,7 @@ def estimate(self):
         # No more than 1GB
         self.config['maintenance_work_mem'] = self.to_mb(self.br((mem / 0x10) > megabytes and megabytes or mem / 0x10))
 
-        self.config['checkpoint_segments'] = 8
         self.config['checkpoint_completion_target'] = '0.7'
-        self.config['wal_buffers'] = self.to_mb(0x200 * self.config['checkpoint_segments'])
```

Comment:
Why you've removed this? Do you know [what `wal_buffers` does](https://www.postgresql.org/docs/9.5/static/runtime-config-wal.html)? You've basically slowed down the entire DB just now. If you removing `checkpoint_segments`, which _is needed_ for <9.6, you should get the average from max/min WAL for>=9.6 instead (see above the another comment).